deployment/docker/Dockerfile.arm64v8

```diff
@@ -0,0 +1,17 @@
+ARG DOTNET_VERSION=3.0
+
+FROM microsoft/dotnet:${DOTNET_VERSION}-sdk as builder
+WORKDIR /repo
+COPY . .
+RUN export DOTNET_CLI_TELEMETRY_OPTOUT=1 \
+ && find . -type f -exec sed -i 's/netcoreapp2.1/netcoreapp3.0/g' {} \; \
```

Comment:
this magic needs a `TODO` or something so we won't forget to either update it when we move to `netcore 2.2` or remove when we move to `3.0` (or when MS would release .net 2.1 for arm64).

Reply:
This should probably be taken care of by drone builds for all platforms.
I will add a TODO as well.